README.md

```diff
@@ -1,28 +1,79 @@
-# Portfolio
+# Portolio-Project
 
-This is a project of my responsive portfolio website that will hold my proffesional information and various projects.
+> Description of the project.
+  
+  This Portolio webpage contain a Complete mobile version and desktop version built with HTML and CSS
+  
+  ## Mobile Version Screenshots
+  
+![Screenshot from 2022-09-30 14-45-48](https://user-images.githubusercontent.com/105572944/193297163-5686f73a-94d3-42c0-bb8c-343c8e1e6920.png)
 
-# To make the portfolio page
+![Screenshot from 2022-09-30 14-46-08](https://user-images.githubusercontent.com/105572944/193297207-32ea268c-b90d-4b07-b06e-ce49c0f406e5.png)
 
-    *using -Html and css
-          -using Vs Code a text editor
-          -github
+![Screenshot from 2022-09-30 14-46-36](https://user-images.githubusercontent.com/105572944/193297225-9720ff1f-36bf-424a-81e4-9f6440f441b9.png)
 
-# Author
+![Screenshot from 2022-09-30 14-49-37](https://user-images.githubusercontent.com/105572944/193297236-1d4a0bc3-254e-4963-aea3-7f484853c601.png)
 
--GitHub":[@mesfin345](https://github.com/mesfin345)
+![Screenshot from 2022-09-30 14-49-54](https://user-images.githubusercontent.com/105572944/193297245-652ecf49-3d0e-4570-8cc9-32e4f7259965.png)
 
-# Acknowledgment
+  
+  ## Desktop Version Screenshots
+  
+![Screenshot from 2022-09-30 14-50-31](https://user-images.githubusercontent.com/105572944/193297326-3418d207-3ab6-49c2-bd4c-552cafa6751a.png)
 
-- for my conding partner
-- my mentor
+![Screenshot from 2022-09-30 14-50-50](https://user-images.githubusercontent.com/105572944/193297335-89dbc5e2-33d7-433b-86a5-cbb812988404.png)
 
-# Link
+![Screenshot from 2022-09-30 14-51-01](https://user-images.githubusercontent.com/105572944/193297352-80f774af-6a6d-44f5-9881-00e55dec77b4.png)
 
-- https://mesfin345.github.io/Portfolio/
+![Screenshot from 2022-09-30 14-51-16](https://user-images.githubusercontent.com/105572944/193297413-f1b22dce-8929-4136-9b07-b34a2bf0c4ed.png)
+
+![Screenshot from 2022-09-30 14-51-40](https://user-images.githubusercontent.com/105572944/193297439-00c30436-0621-4de4-a1f1-3e5b5ee062ea.png)
+
+![Screenshot from 2022-09-30 14-51-30](https://user-images.githubusercontent.com/105572944/193297450-4b826e48-978f-4a6b-8f1a-5a56223a5e86.png)
+
+
+## Built With
+- HTML
+- CSS
+
+
```

Comment:
- [Optional] Please make sure that each of your commit messages should be written in the imperative tense and capitalized. This will make your work look more professional. A good example of a professional commit message is *Add markup to index.html* and a bad example is *added HTML*. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8).

- [Optional] Please in all future projects, make sure that the name of your branch is a feature or an action that is added to the branch not just 'feature'. This is best practice for [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md). A good example of a good branch name is 'headline-section' or 'portfolio-headline'.